posts/Yoann/2018-02-07-hibernate-search-5-9-0-Final.adoc

```diff
@@ -0,0 +1,97 @@
+= Hibernate Search 5.9.0.Final was just released
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published 5.9.0.Final, the first stable release in the 5.9 branch.
+This release brings a brand-new JSR-352 integration for mass indexing,
+WildFly feature packs, and better integration to modular environment.
```

Comment:
I'd make s/modular environment/modular environments/ so be a bit more generic, otherwise you'd need to specifcy which one.